packages/core/src/profile/redux/reducer.js

```diff
@@ -36,6 +36,10 @@ const INITIAL_STATE = {
     error: null,
     isLoading: false,
   },
+  userAttributes: {
+    error: null,
+    isLoading: false,
```

Comment:
You need to store the result of the `getUserAttributes` action in here and not in the main `result` as you are currently doing. Otherwise you will override the profile result that is used by some other selectors, etc. 
Add the `result` property here and update the `reducer` accordingly (see my comment further below in this file).